haskell/private/cabal_wrapper.sh.tpl

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env bash
+#
+# cabal_wrapper.sh <PKG_NAME> <SETUP_PATH> <PKG_DIR> <PACKAGE_DB_PATH> [SETUP_ARGS...]
+
+set -euo pipefail
+execroot=$(pwd)
```

Comment:
```suggestion
execroot="$(pwd)"
```
In principle the path to the execroot could contain whitespace. Granted, a lot of bazel rules would fail in that case. (Same for all other variables below).